app/src/pages/TokenPage/TokenPage.tsx

```diff
@@ -169,12 +173,17 @@ export const TokenPage = (): React.ReactElement => {
             <ResponsiveHidingView hiddenAbove={ScreenSize.Medium}>
               <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
                 <Text alignment={TextAlignment.Center} variant='header1'>{collectionToken.name}</Text>
-                <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
-                  <Text>Owned By</Text>
-                  <Account accountId={owner} />
-                </Stack>
                 {tokenSales && tokenSales.length > 0 && (
-                  <Text>{`Last Bought for Ξ${tokenSales[0].value / 1000000000000000000.0} on ${getTokenDateString(tokenSales[0].blockDate)}`}</Text>
+                  <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
```

Comment:
why do you need a stack here, it's already in a vertical stack. can you use a fragment instead?